MITR Components

ashaw0724 opened the pull request:
DiscussionBox, ReplyBox, and SearchBar components from the MITR repo.

ashaw0724 committed:
MITR Components

DiscussionBox, ReplyBox, and SearchBar components from the MITR repo.

matthewyoungbar commented on src/discussionBox/discussionBox.js:
```diff
@@ -0,0 +1,259 @@
+import { useNavigate  } from "react-router-dom";
+import { useContext, useState} from 'react';
+import Context from '../../../context';
+import { get, post, put } from '../../../util/api';
```
Right now, these components won't work because of their dependencies on outside functions. We can work on making them more generic, but we'll work with what you've provided.

matthewyoungbar reviewed (changes requested):
The components need to be exported in UI.js.
It seems there was a minor styling change you had made to the button as well.

ashaw0724 committed:
UI Exports

Adding the exports to the UI.js file and including the additions to the button styling.
